### PR TITLE
Only print if Verbose is true

### PR DESCRIPTION
**Pull request by xyproto**

### CODE REVIEW

File: bigcache.go [hunk 1]
```diff
@@ -111,7 +111,9 @@ func newBigCache(ctx context.Context, config Config, clock clock) (*BigCache, er
 			for {
 				select {
 				case <-ctx.Done():
-					fmt.Println("ctx done, shutting down bigcache cleanup routine")
```
Comment (cristaloleg) [inline bigcache.go:114]: TBH this can be absolutely dropped, WDYT @janisz ?

Comment (janisz) [reply bigcache.go:114]: I agree

File: bigcache.go [hunk 2]
```diff
@@ -111,7 +111,9 @@ func newBigCache(ctx context.Context, config Config, clock clock) (*BigCache, er
...
+					if config.Verbose {
+						fmt.Println("ctx done, shutting down bigcache cleanup routine")
+					}
```
Comment (janisz) [inline bigcache.go:116]:
```suggestion
```

**Review by cristaloleg — APPROVED** [review]

**Comment by xyproto** [comment]
Cool, thanks for reviewing and merging. 👍

**Pull request merged**